Drop-In/src/androidTest/java/com/braintreepayments/api/SelectPaymentMethodFragmentUITest.kt

```diff
@@ -0,0 +1,236 @@
+package com.braintreepayments.api
+
+import androidx.core.os.bundleOf
+import androidx.fragment.app.testing.FragmentScenario
+import androidx.lifecycle.Lifecycle
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.braintreepayments.api.dropin.R
+import org.hamcrest.Matchers.not
+import org.json.JSONObject
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.util.concurrent.CountDownLatch
+
+// Ref: https://developer.android.com/guide/fragments/test
+@RunWith(AndroidJUnit4::class)
+class SelectPaymentMethodFragmentUITest {
+
+    private lateinit var countDownLatch: CountDownLatch
+
+    private val supportedPaymentMethods = listOf(
+            DropInPaymentMethodType.PAYPAL,
+            DropInPaymentMethodType.PAY_WITH_VENMO,
+            DropInPaymentMethodType.UNKNOWN,
+            DropInPaymentMethodType.GOOGLE_PAYMENT
+    )
+
+    @Before
+    fun beforeEach() {
+        countDownLatch = CountDownLatch(1)
+    }
+
+    @Test
+    @Throws(InterruptedException::class)
+    fun whenStateIsRESUMED_loaderIsVisible() {
+        val scenario = FragmentScenario.launchInContainer(SelectPaymentMethodFragment::class.java)
+        scenario.moveToState(Lifecycle.State.RESUMED)
+        onView(withId(R.id.bt_select_payment_method_loader)).check(matches(isDisplayed()))
```

Comment:
To clarify, this `loader` is a spinner?

Reply:
Yep! The Android equivalent class is `ProgressBar`.